docs/routes.rst

```diff
@@ -0,0 +1,68 @@
+Routes
+======
+
+``rxnutils`` contains routines to analyse synthesis routes. There are a number of readers that can be used to read routes from a number of 
+formats, and there are routines to score the different routes.
+
+Reading
+-------
+
+The simplest route format supported is a text file, where each reaction is written as a reaction SMILES on a line. 
+Routes are separated by comma
```

Comment:
We say that routes are separated by `comma` but below with separate them with a new line.